File: tlg_android/src/ch/bailu/tlg_android/HighscoreActivity.java
package ch.bailu.tlg_android;

import android.app.Activity;
import android.os.Bundle;
import android.text.Html;
import android.widget.ScrollView;
import android.widget.TextView;
import ch.bailu.tlg.HighscoreList;
import ch.bailu.tlg.PlatformContext;

public class HighscoreActivity extends Activity {
	private TextView text;

	@Override
	public void onCreate(Bundle savedInstanceState) {
		super.onCreate(savedInstanceState);
		ScrollView scroll=new ScrollView(this);

		text = new TextView(this);
		text.setTextSize(15f);
		text.setLinkTextColor(new NoDrawContext(this).colorFrame());
		text.setText(Html.fromHtml( getHighscoreText()) );

		scroll.addView(text);
		setContentView(scroll);
	}

	private String getHighscoreText() {
		PlatformContext gc = new NoDrawContext(this);
		HighscoreList list = new HighscoreList(gc);
		return list.getFormatedHTMLText();
	}

}

File: tlg_android/src/main/java/ch/bailu/tlg_android/activity/HighscoreActivity.kt
package ch.bailu.tlg_android.activity

import android.app.Activity
import android.os.Bundle
import android.text.Html
import android.widget.ScrollView
import android.widget.TextView
import ch.bailu.tlg_android.Configuration
import ch.bailu.tlg_android.context.AndroidBaseContext
import context.PlatformContext
import score.HighscoreList

class HighscoreActivity: Activity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        val scroll = ScrollView(this)
        val text = TextView(this)
        text.textSize = 15f
        text.setLinkTextColor(Configuration.frameColor)
        text.text = Html.fromHtml(getHighscoreText())
        scroll.addView(text)
        setContentView(scroll)
    }

    private fun getHighscoreText(): String {
        val bContext: PlatformContext = AndroidBaseContext(this)
        val list = HighscoreList(bContext)
        return list.formatedHTMLText
    }
}
